episodes/chap5_python_client.md

```diff
@@ -35,75 +73,352 @@ server_api = "/api"
+
+::::::::::::::::::::::::
 
-- `client.user`
-- `client.organization`
-- `client.rule`
-- `client.role`
-- `client.collaboration`
-- `client.task`
-- `client.result`
-- `client.util`
-- `client.node`
-- `client.store`
-- `client.algorithm`
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+:::::::::::::::::::::::: solution
+
+1. Make sure you have activated the conda environment with the vantage6 client
+   installed: `conda activate v6-workshop`.
+2. Make sure you have created the `config.py` file with your credentials and the
+   `client.py` file with the code above.
+3. Run the `client.py` script: `python client.py`. If the connection is successful,
+    you should see the message `--> Succesfully authenticated`:
+
+    ```bash
+    (v6-workshop) $ python client.py
+    Welcome to
+                      _                     __
+                     | |                   / /
+    __   ____ _ _ __ | |_ __ _  __ _  ___ / /_
+    \ \ / / _` | '_ \| __/ _` |/ _` |/ _ \ '_ \
+     \ V / (_| | | | | || (_| | (_| |  __/ (_) |
+      \_/ \__,_|_| |_|\__\__,_|\__, |\___|\___/
+                                __/ |
+                               |___/
+
+    --> Join us on Discord! https://discord.gg/rwRvwyK
+    --> Docs: https://docs.vantage6.ai
+    --> Blog: https://vantage6.ai
+    ------------------------------------------------------------
+    Cite us!
+    If you publish your findings obtained using vantage6,
+    please cite the proper sources as mentioned in:
+    https://vantage6.ai/vantage6/references
+    ------------------------------------------------------------
+
+    ...
+
+    --> Succesfully authenticated
+    ```
+
+:::::::::::::::::::::::::::::::::
+
+## Using the client
+
+The `Client` instance offers a set of attributes that correspond to the
+[vantage6 server resources (Episode 3)](./chap3_run_analysis_ui.md#vantage6-user-interface-basics)
+also described in the [official documentation (docs.vantage6.ai)](https://docs.vantage6.ai/en/main/introduction/introduction.html#vantage6-resources). The available attributes are:
+
+| Resource              | Description                                                                 |
+|-----------------------|-----------------------------------------------------------------------------|
+| `client.user`         | Manage users including your own user details                           |
+| `client.organization` | Manage organizations or the organization that you are part of|
+| `client.rule`         | View all available permission rules |
+| `client.role`         | Manage roles (are collections of rules) |
+| `client.collaboration`| Manage collaborations |
+| `client.task`         | Create new tasks and view their run data |
+| `client.result`       | Obtain results from the tasks |
+| `client.util`         | Provides utility functions for the vantage6 Python client. For example to reset your password.             |
```

Comment:
This line is the only one in the table that contains '.' to end a sentence - better to remove them to be consistent